### ISSUE TITLE
Viewer card chat history section

### ISSUE THREAD

**Issue by sh2aliyev** [FrankerFaceZ/FrankerFaceZ#1167]
Showing past messages when the user viewer card is opened.
BetterTV has the feature, but why not in FFZ.

**Comment by Lordmau5** [comment]
From what I know the reason for this is that Twitch natively has an option to see the message history for users if you're a moderator or broadcaster

**Comment by shaanaliyev** [comment]
> From what I know the reason for this is that Twitch natively has an option to see the message history for users if you're a moderator or broadcaster

Yes, Twitch has this feature natively for moderator or broadcaster. But as I said before some other extensions like BTTV offers same feture for normal users too (not all history, just messages collected after you join the chat room).

_FFZ settings » Chat » Viewer Cards » Appearance_
When **"Highlight messages from users with open viewer cards"** section active it offers a use close to what I want, but in general it is very difficult to use in this way.

**Comment by popopoplus** [comment]
I agree, it would be a good addition.
